src/z3c/rml/tests/input/tag-image.rml

```diff
@@ -19,10 +19,5 @@
            x="2in" y="2in" width="0.5in" height="3in"
            doc:example=""/>
 
-    <image file="[z3c.rml.tests]/input/images/replogo.png"
-           x="4in" y="7.3in" height="1in"
-           preserveAspectRatio="true"
-           doc:example=""/>
-
```

Comment:
Your fix for transparent images was basically a quick hack whereas my fix implemented the mask and imageMask attributes as they are in the official RML implementation. I chose my fix (which is tested in tag-image-mask.rml) over yours.